src/librustdoc/html/layout.rs

```diff
@@ -105,7 +105,7 @@ crate fn render<T: Print, S: Print>(
                            placeholder=\"Click or press ‘S’ to search, ‘?’ for more options…\" \
                            type=\"search\">\
                 </div>\
-                <button type=\"button\" class=\"help-button\">?</button>
+                <button type=\"button\" id=\"help-button\">?</button>
```

Comment:
Please declare this ID into `init_id_map` (we should really write a tidy check for that, gonna do that tomorrow
).

Reply:
What does the `IdMap` do?